pkg/base/helm.go

```diff
@@ -74,6 +74,32 @@ func RenderHelm(u *upstreamtypes.Upstream, renderOptions *RenderOptions) (*Base,
 		return nil, errors.Errorf("unknown helmVersion %s", renderOptions.HelmVersion)
 	}
 
+	// add namespace if it's defined in the spec and not already present in the manifests
+	if renderOptions.Namespace != "" {
+		for i, v := range rendered {
+			var manifest map[string]interface{}
```

Comment:
you could use this struct instead: https://github.com/replicatedhq/kots/blob/a7fb3d422442db7094f28bd1788e375adadaa526/pkg/base/base.go#L34

Reply:
I considered that but it omits the spec which means we'd lose it when marshalling. Should I add the spec to that type instead of unmarshalling to `map[string]interface{}`? I'd also worry that it'll omit other things that I haven't thought of yet so `map[string]interface{}` seemed a bit safer.

Reply:
nope, you have a good point ... you can keep it as is